src/audio_generic.cpp

```diff
@@ -22,31 +22,20 @@
 #include <memory>
 #include "audio_decoder_midi.h"
```

Comment:
This line can be deleted if  you move `audio_generic_midiout.h` (see next comment).